src/css/components/_progress.scss

```diff
@@ -15,6 +15,7 @@
 .video-js .vjs-progress-control {
   @include flex(auto);
   @include display-flex(center);
+  margin: 0 0.45em 0 0.45em;
```

Comment:
Looks like this would break the Brightcove skin by making the tooltips go out of bounds with the `keepTooltipsInside`. It probably would also break skins like https://github.com/mmcc/videojs-skin-twitchy.